tests/unit/dojson/test_dojson_hep.py

```diff
@@ -2040,3 +2040,37 @@ def test_title_from_245__a_b():
     result = hep2marc.do(result)
 
     assert expected == result['245']
+
+
+def test_title_from_245_and_247__a_9():
+    schema = load_schema('hep')
+    subschema = schema['properties']['titles']
+
+    snippet = (
+        '<datafield tag="245" ind1=" " ind2=" ">'
+        '  <subfield code="a">C* algebras and differential geometry</subfield>'
+        '</datafield>'
+        '<datafield tag="247" ind1=" " ind2=" ">'
+        '  <subfield code="9">arXiv</subfield>'
+        '  <subfield code="a">C* algebras and differential geometry</subfield>'
+        '</datafield>'
+    )  # record/163205
+
+    expected = [
+        {
+            'title': 'C* algebras and differential geometry',
+        },
+    ]
+    result = hep.do(create_record(snippet))
+
+    assert validate(result['titles'], subschema) is None
+    assert expected == result['titles']
+
+    expected = [
+        {
+            'a': 'C* algebras and differential geometry',
+        },
+    ]
+    result = hep2marc.do(result)
+
+    assert expected == result['245']
```

Comment:
Do we need this case?
As I realized, we don't have dojson rule from json to MARCXML for 247 field.